.babelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "targets": ["last 2 chrome versions", "last 2 firefox versions"],
+  "plugins": [
+    [
+      "@babel/plugin-transform-runtime",
+      {
+        "regenerator": true
+      }
+    ]
+  ],
+  "presets": [
+    "@babel/preset-env",
```

Comment:
This is how we configure preset-env in the profiler:
https://github.com/firefox-devtools/profiler/blob/a299ff00ea661579e96843d4c3d8cc7772133bc5/.babelrc#L4-L13

optional for this patch, it's better to look at this in another PR -- no high priority either.

Reply:
#202